superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndMetricSelect.tsx

```diff
@@ -162,6 +162,7 @@ export const DndMetricSelect = (props: any) => {
 
   const onNewMetric = (newMetric: Metric) => {
     const newValue = props.isMulti ? [...value, newMetric] : [newMetric];
+      props.isMulti || props.multi ? [...value, newMetric] : [newMetric];
```

Comment:
I don't think this line will do anything because the result is not being assigned to a variable.